app_dart/dev/metrics_transfer.dart

```diff
@@ -0,0 +1,220 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Manually transfer Cocoon metrics to Skia perf GCS bucket.
+//
+// The operation is idempotent so you can transfer with the same start date
+// multiple times without having to worrying about duplications.
+
+import 'dart:io';
+
+import 'package:clock/clock.dart';
+import 'package:cocoon_service/src/model/appengine/time_series.dart';
+import 'package:cocoon_service/src/model/appengine/time_series_value.dart';
+import 'package:gcloud/db.dart';
+import 'package:gcloud/storage.dart';
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:intl/intl.dart';
+import 'package:metrics_center/metrics_center.dart';
+
+// The official pub.dev/packages/gcloud documentation uses datastore_impl
+// so we have to ignore implementation_imports here.
+// ignore: implementation_imports
+import 'package:gcloud/src/datastore_impl.dart';
+
+Future<void> main() async {
```

Comment:
can we document how to call this script?

Reply:
Documentations added in https://github.com/flutter/cocoon/pull/1060/commits/aa59bda556377c2e094f58dd9a3c060aac82c65b